infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Orchestrator.java

```diff
@@ -0,0 +1,78 @@
+package org.corfudb.infrastructure.orchestrator;
+
+import io.netty.channel.ChannelHandlerContext;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.infrastructure.IServerRouter;
+import org.corfudb.infrastructure.orchestrator.workflows.AddNode;
+import org.corfudb.protocols.wireprotocol.CorfuMsgType;
+import org.corfudb.protocols.wireprotocol.CorfuPayloadMsg;
+import org.corfudb.protocols.wireprotocol.orchestrator.AddNodeRequest;
+import org.corfudb.protocols.wireprotocol.orchestrator.OrchestratorRequest;
+import org.corfudb.runtime.CorfuRuntime;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Orchestrator is a service that acts on executing actions that belong to workflow instances.
```

Comment:
Who instantiates this class? How is it used?

Reply:
Added more comments.